From eab165e64b121032ba175565afb9b73ce1248b15 Mon Sep 17 00:00:00 2001
From: Debbie Gray <dgray@deborahgraydesign.com>
Date: Tue, 13 Jan 2015 14:24:57 -0800
Subject: [PATCH 1/3] add bottom margin to images

Adds bottom margin to left-aligned or right-aligned images.
---
 style.css | 2 ++
 1 file changed, 2 insertions(+)
 mode change 100755 => 100644 style.css

diff --git a/style.css b/style.css
old mode 100755
new mode 100644
index 9a325f1..3ddb750
--- a/style.css
+++ b/style.css
@@ -544,11 +544,13 @@ a:active {
 	display: inline;
 	float: left;
 	margin-right: 1.5em;
+    margin-bottom: 1.2em;
 }
 .alignright {
 	display: inline;
 	float: right;
 	margin-left: 1.5em;
+    margin-bottom: 1.2em;
 }
 .aligncenter {
 	clear: both;
-- 
1.9.5.github.0

